user-impl/src/main/java/com/example/auction/user/impl/UserRepository.java

```diff
@@ -44,63 +44,45 @@ public UserRepository(CassandraSession session, ReadSide readSide) {
         return countUsers()
-                .thenCompose(
-                        count -> {
-                            int offset = page * pageSize;
-                            int limit = (page + 1) * pageSize;
-                            CompletionStage<PSequence<User>> Users = offset > count ?
-                                    CompletableFuture.completedFuture(TreePVector.empty()) :
-                                    selectUsers(offset, limit);
-                            return Users.thenApply(seq -> new PaginatedSequence<>(seq, page, pageSize, count));
-                        }
-                );
+            .thenCompose(
+                count -> {
+                    int offset = page * pageSize;
+                    int limit = (page + 1) * pageSize;
+                    CompletionStage<PSequence<User>> Users = offset > count ?
+                        CompletableFuture.completedFuture(TreePVector.empty()) :
+                        selectUsers(offset, limit);
+                    return Users.thenApply(seq -> new PaginatedSequence<>(seq, page, pageSize, count));
+                }
+            );
     }
 
     public CompletionStage<Integer> countUsers() {
         return session
-                .selectOne(
-                        "SELECT COUNT(*) FROM UserInfo  "
-
-                )
-                .thenApply(row -> (int) row.get().getLong("count"));
+            .selectOne(
+                "SELECT COUNT(*) FROM UserInfo  "
+            )
+            .thenApply(row -> (int) row.get().getLong("count"));
     }
 
     private CompletionStage<PSequence<User>> selectUsers(long offset, int limit) {
-
         return session
-                .selectAll(
-                        "SELECT * FROM UserInfo  " +
-                                "LIMIT ?",
-
-                        limit
-                )
-                .thenApply(List::stream)
-                .thenApply(rows -> rows.skip(offset))
-                .thenApply(rows -> rows.map(UserRepository::convertUserSummary))
-                .thenApply(UserSummaries -> UserSummaries.collect(Collectors.toList()))
-                .thenApply(TreePVector::from);
-    }
-
-    protected CompletionStage<UUID> getUserIdByEmail(String email) {
-
-        return session
-            .selectOne(
-                "SELECT * FROM UserInfo  WHERE email = ? " ,
-                   email
+            .selectAll(
+                "SELECT * FROM UserInfo  " +
+                    "LIMIT ?",
+                limit
             )
-
-            .thenApply(rows -> rows.map(UserRepository::convertUserSummary).get())
-            .thenApply(UserSummary -> UserSummary.getId());
-
-
+            .thenApply(List::stream)
+            .thenApply(rows -> rows.skip(offset))
+            .thenApply(rows -> rows.map(UserRepository::convertUserSummary))
+            .thenApply(UserSummaries -> UserSummaries.collect(Collectors.toList()))
+            .thenApply(TreePVector::from);
```

Comment:
This whole block above is a bit messy on the GH UI. There are two changes: `selectUsers` was formatted and `getUserIdByEmail` was removed.

(using `?w=1` in the URL won't let me add review comments :-( )

Reply:
I always end up opening the `w=1` URL in a separate browser tab so I can flip back and forth for this reason.

Reply:
Dear Github,

this is no working. Plesae fix ASAP as we needs for going to real productionz.